### DISCUSSION TITLE
Do you have the ui design kit in Figma or similar?

### DISCUSSION THREAD

**Discussion by VanishMax** [element-plus/element-plus#3191] [Q&A] [answered]
Our company's designers are willing to make their layouts based on Element-ui component styles. And the ui kit would make it way easier. They don't care about whether it's the Sketch or Figma files, just anything. So, do you have something similar?

**Comment by purepear** [comment, marked as answer]
There is https://element-plus.org/#/en-US/resource but i'm not sure how up to date it is.

**Comment by VanishMax** [comment]
Hey @purepear! Thank you for the answer, I haven't noticed this before.
So, I've used the provided .sketch file, converted it to Figma and, I think, it would be useful for many people. Leaving the file here, maybe someone will put it into the docs: https://drive.google.com/file/d/1OVpqkrZI9Kj6PajLmqo8yFlMzEs0PEWh/view?usp=sharing.

And yes, the design does not contain many components that are in the Element-ui. If my team has time, we could improve it and share with you!

**Comment by jw-foss** [comment]
@VanishMax @purepear We will have that when our designer gets time to do that, and we are planning to make that maybe in EAT Nov maybe? Sorry for the delay :bow:
